### PR TITLE
QA-1952: Make Wiki page for test runner framework.

### PR DESCRIPTION
**Pull request by ichengchang**
The Wiki page (first draft) is now up. 

[https://github.com/DataBiosphere/terra-test-runner/wiki](https://github.com/DataBiosphere/terra-test-runner/wiki)

The [Quick Start ](https://github.com/DataBiosphere/terra-test-runner/wiki/Quick-Start) page in this draft shows how to integrate `Test Runner Framework` with `Scala` repo, complete with examples of porting existing `swat` tests into `Test Runner`. 

There are several `Gradle` repos already using `Test Runner` that provide great examples so it's less of a priority to write a formal Wiki for `Gradle` in this first draft.

`Maven` section will be added when we have time.

### CODE REVIEW

File: wiki/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,1567 @@
...
+| Gradle  | <pre>dependencies {<br>  ext {<br>    testRunnerVersion = "0.1.6-SNAPSHOT"<br>  }<br>  implementation "bio.terra:terra-test-runner:${testRunnerVersion}"<br>}</pre> | <pre>plugins {<br>  id 'bio.terra.test-runner-plugin'<br>}</pre> |
+| Sbt     | <pre>val testRunnerVersion = "0.1.6-SNAPSHOT"<br>libraryDependencies += "bio.terra" % "terra-test-runner" % testRunnerVersion</pre> | Plugin unavailable |
+
+The `Test Runner Framework` contains a `Gradle Plugin` implementation to provide a succinct way of launching useful tasks when using `Gradle` build tool CLI. Plugins for other build tools will be available in the future. The `Test Runner Framework Plugin` supports the following tasks
+
+| Task Name | Description | Gradle | Maven | Sbt |
+|-----------|-------------|--------|-------|-----|
+| PrintHelp | Output available task list to console | <pre>./gralew printHelp</pre> | <pre>mvn compile exec:java -Dexec.mainClass="bio.terra.testrunner.common.commands.PrintHelp"</pre> | <pre>sbt runMain bio.terra.testrunner.common.commands.PrintHelp</pre> |
```
Comment (zloery) [inline wiki/quickstart.md:1563]: nit: gralew -> gradlew

**Review by zloery — APPROVED** [review]
This is awesome, thank you! I'm not as familiar with some of the Broad's existing test infrastructure so I can't comment much on the Scala quickstart example, but this looks like it'll be a great resource for any teams writing TestRunner tests for Scala services!

File: wiki/quickstart.md (hunk 1 above)
Comment (ichengchang) [reply wiki/quickstart.md]: Great catch! Fixed typo.

File: wiki/home.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+* Integration
+* Performance
+* Resiliency
+
+The independent release test types and their definitions are still being actively discussed in `Independent Releases & Testing Steering Group Sync`.
+
+# Developers Guide
+
+* [[Quick Start|Quick Start]]
```
[the remark this answers is not in the document]
Comment (ichengchang) [reply wiki/home.md]: Replaced by actual link to quick start page.

**Comment by ichengchang** [comment]
@zloery Thanks for reviewing this!

**Pull request merged**